### ISSUE TITLE
Add code coverage to PyWebAssembly

### ISSUE THREAD

**Issue by axic** [ewasm/testnet#59]
PyWebAssembly follows the wasm spec very closely, so by adding code coverage to it and running only the official wasm test suite should give very accurate report of official test coverage.

**Comment by poemm** [comment]
It seems that coverage.py is popular. I will look into this.

**Comment by chfast** [comment]
Yes, use coverage package to get coverage report. Then upload to Codecov with codecov package.

**Comment by chfast** [comment]
https://github.com/codecov/example-python
